src/app/ui-kit/layout/header/header.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-ui-header',
+  templateUrl: './header.component.html',
+})
+export class HeaderComponent {
+  logoSrc: string = '../../../..//assets/images/main-logo.svg';
```

Comment:
To make your life and the code easier in terms of asset path you can try to update the angular config.
Example
          "options": {
            "assets": [
              "applications/admin/src/favicon.ico",
              "applications/admin/src/manifest.json",
              "applications/admin/src/assets",
              {
                "glob": "**/*",
                "input": "shared/ui/assets",
                "output": "assets/"
              }
            ],
            
            
Input is a path to your asset folder from root
Output - desired path for usage